indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -134,3 +136,20 @@ class RHRoomSuggestions(RHRoomBookingBase):
     @use_args(search_room_args)
     def _process(self, args):
         return jsonify(get_suggestions(args, limit=NUM_SUGGESTIONS))
+
+
+class RHBookingDetails(RHRoomBookingBase):
+    def _process_args(self):
+        self.booking = Reservation.get_one(request.view_args['booking_id'])
+
+    def _process(self):
+        attributes = reservation_details_schema.dump(self.booking).data
+        date_range, occurrences = get_booking_occurrences(self.booking)
+        date_range = [dt.isoformat() for dt in date_range]
+        occurrences = {dt.isoformat(): reservation_details_occurrences_schema.dump(data).data
+                       for dt, data in occurrences.iteritems()}
+        edit_logs = sorted(reservation_edit_log_schema.dump(self.booking.edit_logs).data,
+                           key=itemgetter('timestamp'),
+                           reverse=True)
+        return jsonify(attributes=attributes, id=self.booking.id, occurrences=occurrences, date_range=date_range,
```

Comment:
what about `booking=attributes` - in the end it's the basic booking data we have here.

Or, crazy idea, do we even need this separate? ie serialize the booking details, then add the other information to that dict.
On the Python side we have all on the same "layer" as well (booking.id, booking.occurrences, etc.)